test/wsteth-price-feed.ts

```diff
@@ -0,0 +1,77 @@
+import hre, { ethers } from 'hardhat';
+import { exp, expect } from './helpers';
+import { HttpNetworkConfig } from 'hardhat/types/config';
+import {
+  SimplePriceFeed__factory,
+  SimpleWstETH__factory,
+  WstETHPriceFeed__factory
+} from '../build/types';
+
+export async function makeWstETH({ stEthPrice, tokensPerStEth }) {
+  const SimplePriceFeedFactory = (await ethers.getContractFactory('SimplePriceFeed')) as SimplePriceFeed__factory;
+  const stETHpriceFeed = await SimplePriceFeedFactory.deploy(stEthPrice, 8);
+
+  const SimpleWstETHFactory = (await ethers.getContractFactory('SimpleWstETH')) as SimpleWstETH__factory;
+  const simpleWstETH = await SimpleWstETHFactory.deploy(tokensPerStEth);
+
+  const wstETHPriceFeedFactory = (await ethers.getContractFactory('WstETHPriceFeed')) as WstETHPriceFeed__factory;
+  const wstETHPriceFeed = await wstETHPriceFeedFactory.deploy(
+    stETHpriceFeed.address,
+    simpleWstETH.address
+  );
+  await wstETHPriceFeed.deployed();
+
+  return {
+    simpleWstETH,
+    stETHpriceFeed,
+    wstETHPriceFeed
+  };
+}
+
+const testCases = [
+  {
+    stEthPrice: exp(1000, 8),
```

Comment:
just added some easily verifiable price test cases, e.g.:

if the price of `stETH / USD` is $1000, and one `wstETH` is worth .2 `stETH`, then the `wstETH / USD` price should be $200.

suggestions for additional test cases welcome. might be worth doing some much larger and small numbers, though at some point we're just testing that Solidity's multiplication works as expected.

Reply:
It's actually the other way around. If `tokensPerStEth=0.2`, it means that one `wstETH` is worth `5 stETH`. So the price of `wstETH` should be $5000.

Reply:
The [description](https://etherscan.io/token/0x7f39c581f595b53c5cb19bd0b3f8da6c935e2ca0#readContract) of the `tokensPerEth` function is:

`Get amount of wstETH for one stETH`

The value right now is `916820579837578359`, so `916820579837578359` / 1e18 = 0.9168205798375784.

My reading of that is that it means that one `wstETH` is currently worth ~.91 `stETH1`

Reply:
There's a different function, `stEthPerToken` which I believe would return the value you're describing: `Get amount of stETH for one wstETH`

Reply:
updated